### ISSUE TITLE
Additional tag can prevent searching

### ISSUE THREAD

**Issue by AntonM030481** [organicmaps/organicmaps#2717]
E.g. here all showers are recognized as showers, and have appropriate text. But only one of them is searchable. One with amenity=shower, but without shower=outdoor.
35.331222, 33.419358
![image](https://user-images.githubusercontent.com/85622715/173223823-c9866237-2bcf-493d-b8a4-6347b162cd2f.jpeg)

**Comment by biodranik** [comment]
@vng can you please take a look? Showers are really important in some places like New Zealand )

**Comment by vng** [comment]
Looks like its a search issue when removing duplicating results (with some distance threshold). We alrwady have similar issues like https://github.com/organicmaps/organicmaps/issues/2070

**Comment by biodranik** [comment]
Wow, if it's a duplicate filter, it should not filter out different features visible on the screen, obviously.

**Comment by AntonM030481** [comment]
@vng you are right. If search from shifted position - result will be different (other showers will be filtered out)

**Comment by AntonM030481** [comment]
Verified
